src/matrix.jl

```diff
@@ -16,10 +18,33 @@ function sub(x::_MatTypes, r1::Int, c1::Int, r2::Int, c2::Int)
   return deepcopy(view(x, r1:r2, c1:c2))
 end
 
-function sub(x::_MatTypes, r::AbstractUnitRange{Int}, c::AbstractUnitRange{Int})
+function sub(x::__MatTypes, r::AbstractUnitRange{Int}, c::AbstractUnitRange{Int})
   return deepcopy(view(x, r, c))
```

Comment:
This change is removing functionality for `FqMatrix`. I think that this change (and then also the definition of `__MatTypes` can just go (as it does not do anything anymore